src/app/utils/open-api-parser.ts

```diff
@@ -0,0 +1,50 @@
+export function parseOpenApiResponse(params: any) {
+  const { options: { paths }, url } = params;
+
+  try {
+    const parameters: any[] = [];
+    const requestUrl = Object.keys(paths)[0];
+    const verbs = Object.keys(paths[`${requestUrl}`]);
+    const pathValues: any = Object.values(paths)[0];
+
+    verbs.forEach((verb: string) => {
+      parameters.push({
+        verb,
+        values: getVerbParameterValues(pathValues[`${verb}`]),
+        links: getLinkValues(pathValues[`${verb}`])
+      });
+    });
+
+    return { url, parameters };
+  } catch (error) {
+    return { error };
+  }
+}
+
+function getVerbParameterValues(values: any) {
```

Comment:
same here. The function assumes that `values` has a well defined type and we should reflect that in the input type instead of `any`.